src/test/java/org/cactoos/io/HeadInputStreamTest.java

```diff
@@ -59,11 +64,12 @@ public void testSkippingMoreThanTotal() throws Exception {
             5
         );
         stream.skip(7L);
-        MatcherAssert.assertThat(
+        final String input = new TextOf(stream).asString();
+        new Assertion<>(
             "The result text wasn't empty",
-            new TextOf(stream),
-            new TextHasString("")
-        );
+            () -> new TextOf(input),
```

Comment:
@Iprogrammerr put `new TextOf(stream).asString()` here, instead of create a local variable.